src/dotnet/ToolPackage/IToolPackageStore.cs

```diff
@@ -8,18 +8,8 @@
 
 namespace Microsoft.DotNet.ToolPackage
 {
-    internal interface IToolPackageStore
+    internal interface IToolPackageStore: IToolPackageStoreQuery
```

Comment:
I don't agree with this refactoring.  These are all methods relating to the management and inspection of the package store.  If anything, the users of `IToolPackageStore` after this refactoring are only interested in what should be the "query" part of the interface (i.e. they only care about what's in the store, but not the management methods used by the installer and uninstaller).  Thus this seems backwards to me because the `IToolPackageStoreQuery` interface is more about management of the store and the `IToolPackageStore` interface has both query and management.

Stepping back, is there really a need for this?  All of these methods now relate to the store, albeit some are used to query the store and some are used by the install/uninstaller for managing the store.  This refactoring seems only intended to further abstract what should be possible to do from the installer and uninstaller and I don't feel that's necessary.

Reply:
summery, make them separate interface and swap the 2 names